Artesian/Artesian.SDK/Dto/DeleteCurveData/DeleteCurveData.cs

```diff
@@ -56,7 +56,7 @@ public DeleteCurveData(MarketDataIdentifier id, LocalDateTime version)
         public LocalDateTime? Version { get; set; }
 
         /// <summary>
-        /// The timezone. Must be the OriginalTimezone or, when Hourly, must be "UTC".
+        /// Timezone of the delete range. For DateSerie must be the OriginalTimezone of the Serie. In case null or empty the default is CET for TimeSerie and OriginalTimezone for DateSerie
```

Comment:
"For DateSerie if provided must be equal to MarketData OrignalTimezone Default:MarketData OrignalTimezone.
For TimeSerie Default:UTC  "